targets/targets.json

```diff
@@ -4008,5 +4008,49 @@
         "detect_code": ["7013"],
         "release_versions": ["5"],
         "bootloader_supported": true
+    },
+    "ARM_FM": {
+        "inherits": ["Target"],
+        "public": false,
+        "extra_labels": ["ARM_FM"]
+    },
+    "FVP_MPS2": {
+        "inherits": ["ARM_FM"],
+        "public": false,
+        "supported_toolchains": ["GCC_ARM", "ARM", "IAR"],
+        "extra_labels_add": ["FVP_MPS2"],
```

Comment:
Redundant. This already happens because of the target name.

Reply:
Didn't realize that was automatically added. Now got all of these redundant "extra_label_add" removed. can you review pls @theotherjimmy